util/slack.q

```diff
@@ -21,13 +21,15 @@ taguser:{[uid] "<@",uid,">"}
 /-- api calls --
 /the following are wrappers around slack API calls (of the same name) exposing very basic functionality
 /for more advanced usage of these API calls, more sophisticated code will be necessary
-channels.list:{.Q.hp[url`channels.list;.post.ty`form;.post.urlencode (1#`token)!enlist token]}
-channels.info:{[chid].Q.hp[url`channels.info;.post.ty`form;.post.urlencode `token`channel!(token;chid)]}
+conversations.list:{[types].Q.hp[url`conversations.list;.post.ty`form;.post.urlencode (`token`limit`exclude_archived`type)!(token;500;"true";types)]}
+conversations.info:{[chid].Q.hp[url`conversations.info;.post.ty`form;.post.urlencode `token`channel!(token;chid)]}
+channels.list:conversations.list["public_channel"]
```

Comment:
if I'm not missing something, this used to be a function and will now be a dictionary? think that could break stuff elsewhere, might need to make `conversations.list` take two args & make this (and `groups.list` & `im.list`) a projection?

Reply:
you right, but don't think we need the extra arg as it wasn't used in the old calls. made them lamdbas so should play nicely now

Reply:
also worth noting that in the return of groups.list the response has `channels` as the key of the dict not `groups`, fairly sure that i removed all the parts where groups was referred to directly, but not 100% certain